adhocracy4/comments_async/serializers.py

```diff
@@ -47,6 +50,9 @@ def to_representation(self, instance):
                 else:
                     categories[category] = category
         ret['comment_categories'] = categories
+        is_blocked = ret.get('is_blocked')
+        if is_blocked:
+            ret.__setitem__('comment', '')
```

Comment:
Why not `ret['comment'] = ''`?

Reply:
I refactored this in the last commit. Thx!